charts/delphai-deployment/values.yaml

```diff
@@ -143,6 +143,8 @@ ingress:
 rabbitmq:
   enable: false
   
+  maxReplicaCount: 16
```

Comment:
Max count added in d572a1bb9b9a7645a53da5d39afd2c60109e7c81

Currently the production cluster has only one service with more than 16 replicas: the `page-scraper` has 30. Both `company-pages` and `data-processing` have 16. This seems like a decent default value to try.

Reply:
Yeah, nice default